oat-sa/tao-core
# Feature/aut 3629/solar folder structure

related to https://oat-sa.atlassian.net/browse/AUT-3629

### Description

This PR contains backward-compatible changes of `jsTree` library to be able to implement design upon the spec

<img width="533" alt="Screenshot 2024-05-10 at 14 22 56" src="https://github.com/oat-sa/tao-core/assets/10635482/fb41fc29-a05e-4fe2-902d-b7b93f89ce16">

Some of icons do not exist in mockups.
default were used for :

`icon-delivery`, `icon-test-taker`, `icon-users`, `icon-table`

### How to test

 - check out the branch: git checkout -t origin/feature/AUT-3629/solar-folder-structure
 - activate the feature flag: php index.php 'oat\tao\scripts\tools\FeatureFlag\FeatureFlagTool' -s FEATURE_FLAG_SOLAR_DESIGN_ENABLED -v true
 - log in to TAO backoffice
 - look at the tree structure in Items, Tests, ...
